### DISCUSSION TITLE
Unable to get a response from an API, Trying to get access_token

### DISCUSSION THREAD

**Discussion by Sagar409-conflict** [keycloak/keycloak#35695] [Q&A]
My newly created realm name is `Test-Realm` I also created a new user inside that by providing the  necessary information and attaching a password for it though I'm unable to get the token through Postman

How can I get `access_token ` form the API Response?

For this Endpoint:
`http://localhost:8080/realms/Test-Realm/protocol/openid-connect/token`

Body:
```
{
"client_id": "Test-Client"
"client_secret":"Tl0S***************"
"username": "root"
"password": "temp123"
"grant_type ": "password"
}
```

Response:
```
{
    "error": "invalid_grant",
    "error_description": "Account is not fully set up"
}
```

**Comment by embesozzi** [comment]
Check if the user `root` in the realm `Test-Realm` has any required actions assigned. If the user has any pending required actions, I believe the ROPC will fail because the account is not fully set up.

**Reply by Sagar409-conflict** [reply]
So now what's happening is, that I've configured KeyCloak, and I have one protected route in my API code, I want KeyCloak to redirect me to their Login Page (as I've chosen "keycloak" as the login theme), But I'm getting this page from their side,
 
on this URL: `http://localhost:8080/auth/realms/AmpliGuide/protocol/openid-connect/auth?client_id=ampliguide-log&state=66c15010-8c67-47ff-8844-5fcd1b1fb44c&redirect_uri=http%3A%2F%2Flocalhost%3A8000%2F%3Fauth_callback%3D1&scope=openid&response_type=code`


![image](https://github.com/user-attachments/assets/afed8ece-a879-4231-b02d-d64b85c766ce)


How can I get the Login Page provided by Keycloak?

**Reply by danielFesenmeyer** [reply]
Are you sure that the realm `AmpliGuide` exists?
